onnxruntime/python/tools/transformers/onnx_exporter.py

```diff
@@ -270,12 +270,17 @@ def load_pt_model(model_name, model_class, cache_dir):
 
 def load_tf_model(model_name, model_class, cache_dir):
     config = AutoConfig.from_pretrained(model_name, cache_dir=cache_dir)
-
+    
+    # Loading tf model from transformers limits the cpu affinity to {0} when KMP_AFFINITY is set
+    # Restore the affinity after model loading for expected ORT performance
+    pid = os.getpid()
+    affinity = os.sched_getaffinity(pid)
```

Comment:
FYI, this is only available for linux. This function doesn't exist for Mac.
```python
>>> os.sched_getaffinity(pid)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'os' has no attribute 'sched_getaffinity'
```

Reply:
Oh thanks for the info. seems this is only available on part of Unix. On windows there is another story.